pkg/controllers/state/suite_test.go

```diff
@@ -1832,9 +1834,10 @@ var _ = Describe("Taints", func() {
 			ExpectReconcileSucceeded(ctx, nodeController, client.ObjectKeyFromObject(node))
 
 			stateNode := ExpectStateNodeExists(cluster, node)
-			Expect(stateNode.Taints()).To(HaveLen(3))
```

Comment:
Can we add a test to ensure the schedulability of pods that don't tolerate this taint while the node is uninitialized

Reply:
Added